turbo/snapshotsync/snapshots.go

```diff
@@ -51,6 +51,7 @@ import (
 type SortedRange interface {
 	GetRange() (from, to uint64)
 	GetType() snaptype.Type
+	GetTypeString() string
```

Comment:
this seems like a property of the Type, not of the SortedRange.

Reply:
Transaction is a `Type`. It has two indexes within it, both of which need to be differentiated.

Reply:
in new push, added a "Grouping", removed "Type"